LISAv2-Framework.psm1

```diff
@@ -160,7 +161,7 @@ function Start-LISAv2 {
 			}
 
 			# Run test
-			$testController.RunTest($TestReportXml,$TestIterations,$false)
+			$testController.RunLoadedTestCases($TestReportXml, $TestIterations, $false)
```

Comment:
In above code, there is a function "LoadTestCases". So here rename this function to "RunLoadedTestCases" makes more sense.